iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/model/HouseTable.java

```diff
@@ -39,4 +39,6 @@ public class HouseTable {
   private long lastModifiedTime;
 
   private long creationTime;
+
+  private String storageType;
```

Comment:
We don't have a lot of comments here describing the fields. At least this one deserves some explanation with pointer to the list of supported types.

Reply:
Also, has this column been already added to the schema of all deployed databases?

Reply:
this column has been added to the deployed database schema already